projects/Omnom/default.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+
+  metadata = {
+    summary = "Omnom is a webpage bookmarking and snapshotting service.";
+    subgrants = [
+      "Omnom"
+      "Omnom-ActivityPub"
+    ];
+  };
+
+  nixos.modules.services = {
+    Omnom = {
```

```suggestion
    omnom = {
```

While we're really flexible with calling our attributes whatever we like, and while it surely makes sense to use exactly the project name, the NixOS convention is to keep it in lower case (which also happens to be ever so slightly simpler to type), so I suggest we follow that to avoid surprise.